.changeset/swift-pumpkins-shake.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/backend-app-api': minor
+'@backstage/backend-common': minor
+'@backstage/config-loader': minor
+'@backstage/plugin-app-backend': minor
+'@backstage/backend-dynamic-feature-service': minor
```

Comment:
Could you split this up in a couple more changesets that more accurately describe the changes to each package?

Reply:
Done in commit 7f74e6409bf781eb248c8392cf19e7f58a90aab9